src/main/kotlin/com/sourcegraph/cody/agent/protocol/GetCurrentUserCodySubscription.kt

```diff
@@ -0,0 +1,18 @@
+package com.sourcegraph.cody.agent.protocol
+
+object GetCurrentUserCodySubscription {
```

Comment:
We can consider renaming this class.

We should consider options to automatically generate the values of the enums. 

However, I think we can simply remove what is redundant now. It will be no worse. Then, having less code it will be easier to improve the generation.

Reply:
I think we should put it in `protocol_extensions` package.

Reply:
makes sense ! thanks

Reply:
fixed!